TeamMandate/Mandate.md

```diff
@@ -0,0 +1,53 @@
+# IT Strategy 
+
+## Vision  
+
+Create the Canadian Experience by leveraging modern technologies where government interacts seemlessly with Canadians through existing digital infrastructure 
+
+## Mission 
+
+Be the initiator of change within IITB to lay the path for the evolution of ESDCs digital services through an open collaborative approach to further ESDCs mandate in the digital era by alinging IITB with TBS Digital Standards 
+
+## Mandate 
+
+- Provide guidance to senior management on how IT can be leveraged to allow ESDC to deliver on its mandate 
```

Comment:
Change "provide guidance" with "Provide direction"
Or if direction is too strong, "Provide recommendations"